gadgets/trace_exec/gadget.yaml

```diff
@@ -21,6 +21,12 @@ datasources:
       comm:
         annotations:
           template: comm
+      cwd:
+        annotations:
+          description: The current working directory of the executable
+          columns.width: 64
+          columns.hidden: "true"
```

Comment:
Instead of having a literal like "true"/"false" for `columns.hidden` it could be nice to have it based on ebpf param paths. Something like `columns.hidden: "param:paths"`.

I think it's not supported right now [here](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/datasource/columns.go#L81) but could be nice to have, in addition to the discussion about optional fields